docs/faq/network/validator.md

```diff
@@ -0,0 +1,114 @@
+
+In the future we aim to work with third party services to make setting up solo validator nodes easier with minimal setup. However, this will not be available at genesis as the network will need to be running for some time before third parties are able to integrate.
+
+### What penalties will I receive if my node goes offline?
+
+A node going offline is usually not a slashable offense. However, your stake will be penalized by around the same amount as the rewards you would have gained were the node online.
+
+### How can I arrange my taxes if I receive validator rewards?
+
+This is not something the LUKSO team can help with. You are responsible for researching your local tax jurisdiction and making sure you are compliant with your local tax regulations. There are many tax tools and services that were built for Ethereum validators that can also be helpful for LUKSO validators.
+
+### Are nodes operated through a CLI or will there also be a GUI option?
+
+Running a validator node requires a degree of technical proficiency and as such those looking to run a validator should be comfortable using the terminal to use the LUKSO CLI. There will also be a launchpad website which will streamline the process of generating deposit data and transferring validator funds, though the node setup and maintenance will require using the [LUKSO CLI](https://github.com/lukso-network/tools-lukso-cli).
+
+### What will be the different validator clients?
+
+At launch, the LUKSO CLI is able to install the following clients for running a validator node. More will become available in the future.
+
+- Execution Clients: [Geth](https://geth.ethereum.org/)
+- Consensus Clients: [Prysm](https://github.com/prysmaticlabs/prysm)
+
+### How can I set my withdrawal address for my validators rewards after the Shapella upgrade?
+
+For details on how to enable withdrawals after the Shapella upgrade see these guides:
+
+- [https://notes.ethereum.org/@launchpad/withdrawals-guide](https://notes.ethereum.org/@launchpad/withdrawals-guide)
+- [https://docs.prylabs.network/docs/wallet/withdraw-validator](https://docs.prylabs.network/docs/wallet/withdraw-validator)
+
+### Will lower latency positively influence my rewards?
+
+When the appointed time for a validator comes to propose a block the validator has 12s to vote. 2 scenarios could cause a validator to miss that slot:
+
+1. The node is not synced up to the canonical head of the network.
+2. The validator needed more than 12s to propose the block.
+
+There are no extra rewards for being very quick.
+
+### How can I set up my node to maximize my peers?
+
+In the normal case a node will connect to the maximum peers. Each new peer connection goes through an evaluation step that will judge if a node is suitable, otherwise it will disconnect. You can check the logs to see disconnection from peers.
```

Comment:
```suggestion
Typically, a node will connect to the maximum number of peers. Each new peer connection goes through an evaluation step to judge if a node is suitable. Otherwise, it will disconnect. You can check the logs to see disconnection from peers.
```